app/boards/arm/adv360pro/adv360pro-layouts.dtsi

```diff
@@ -0,0 +1,87 @@
+#include <physical_layouts.dtsi>
+
+/{
+    default_layout: default_layout {
```

Comment:
I am on the fence on the naming convention here of `default_layout`. Not opposed, just pondering. Some other ideas:

* `standard_layout`
* `std_layout`
* `stock_layout`
* `physical_layout`
* `physical_layout0`

Thoughts?

Reply:
`physical_layout0` would be the only contender aside from `default_layout` in my eyes.

Reply:
Yeah, that matches the `kscan0` convention we've used, thus my suggestion. @joelspadin or @caksoylar any opinions?

Reply:
I like `physical_layout0` also, since it is consistent with kscan (but breaks the matrix transform convention).

Reply:
Changed everything over to `physical_layout0`. Also renamed some transform labels to `matrix_transform0` for consistency.

Reply:
Sorry to bring this up after the changes, but I think we should add the new labels as additional rather than replace the existing ones. Othewrise we'll break existing configs which might use them in `chosen` nodes.

Reply:
I only changed the ones where we only have a single physical layout defined, or where the changed label is already the chosen one. I can't think of another reason why someone would need to refer to a physical layout or matrix transform in their config.